testmodule/helpers.nim

```diff
@@ -0,0 +1,53 @@
+import std/json
```

Comment:
If you want this to be available to users of the library, then you'd have to put in a file under `ethers/`, e.g. `ethers/testing.nim`. It can then be imported using `import pkg/ethers/testing`.

Reply:
1. Changed to `ethers/testing.nim`
2. Renamed `testmodule/testHelpers.nim` to `testmodule/testTesting.nim` and imported `pkg/ethers/testing`.